src/Caller.cpp

```diff
@@ -709,11 +720,189 @@ bool Caller::parseOptions(int argc, char **argv) {
   }
   // if there is more then one argument left...
   if (cmd.arguments.size() > 1) {
-    cerr << "Error: too many arguments.";
-    cerr << "\nInvoke with -h option for help\n";
-    return 0; // ...error
-  }
+    bool allPin = true;
+    for(const string &text : cmd.arguments) {
+      if(text.substr(text.find_last_of(".") + 1) != "pin") {
```

Comment:
Is this really neaded? We normally do not require an extension pin. E.g. our test code uses an extension .tab for the same file format. Originally the .pin was referred to the percolator XML format. This is, however, something that we have not enforced. Its there an other way to check if these are tab delimited?